gossip/evmstore/store_test.go

```diff
@@ -1,6 +1,11 @@
 package evmstore
 
 import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/assert"
```

Comment:
It will be better to use "github.com/stretchr/testify/require" instead, for uniformity purpose.

Reply:
Make sense, changed